app/modules/intelligence/provider/provider_service.py

```
@@ -44,319 +61,557 @@ async def list_available_llms(self) -> List[ProviderInfo]:
-                    "Development mode enabled. Using environment variable for API key."
-                )
-                openai_key = os.getenv("OPENAI_API_KEY")
-                if agent_type == AgentType.CREWAI:
-                    self.llm = LLM(
-                        model="openai/gpt-4o-mini",
-                        api_key=openai_key,
-                        temperature=0.3,
-                    )
+            if preferred_provider == "openai":
+                logger.info("Initializing OpenAI LLM")
+                if os.getenv("isDevelopmentMode") == "enabled":
```

Ruff (0.8.2)

Use capitalized environment variable `ISDEVELOPMENTMODE` instead of `isDevelopmentMode`

(SIM112)